src/types.lua

```diff
@@ -38,16 +38,17 @@
 
 ---@class lest.TestResultNode
 ---@field type lest.NodeType
+---@field pass boolean
 ---@field name string
 
 ---@class lest.TestResult: lest.TestResultNode
 ---@field type 0
----@field pass boolean
 ---@field error? any
 
 ---@class lest.DescribeResults: { [number]: lest.TestResult | lest.DescribeResults }
 ---@field type 1
 ---@field name string
 
 ---@class lest.TestSuiteResults: lest.DescribeResults
+---@field pass boolean
```

Comment:
This is already declared on the base `lest.TestResultNode` class

Reply:
![a](https://i.imgur.com/iAld0xl.png)
Doesn't seem to work? This is why I added that pass field.

Reply:
DescribeResults needs to use multiple inheritance